app/views/likes/create.js.erb

```diff
@@ -1,5 +1,7 @@
-$('#like').bind('ajax:success',function() {
+$('.btn').bind('ajax:success',function() {
   $(this).closest('div').find('span').html(<%= "#{@post.likes.count}" %>);
+  $(this).closest('div').find('a').html("Liked");
+  $(this).attr("href", "/posts/" + <%= "#{@post.id}" %>+"/likes/"+<%= "#{@like.id}" %>)
```

Comment:
This was old approach and unstable way to concat strings. 
use ES6 approch. 
e.g 
```
let firstName = "Talha" ; let lastName = "Ali"
let fullName=`${firstName} ${lastName}`
```

Reply:
Done